config/database.yml

```diff
@@ -1,6 +1,8 @@
 development:
   adapter: postgresql
-  database: discourse_development
+  database: <%= ENV['DISCOURSE_DB_NAME'] || 'discourse_development' %>
```

Comment:
Hmm... I think it could be changed so that in any mode, it uses what it finds in the env.